std/algorithm/comparison.d

```diff
@@ -36,6 +36,12 @@ $(T2 mismatch,
 $(T2 predSwitch,
         $(D 2.predSwitch(1, "one", 2, "two", 3, "three")) returns $(D "two").)
 )
+$(T2 isSameLength,
+        $(D isSameLength([1, 2, 3], [4, 5, 6])) returns $(D true).)
+)
+$(T2 isPermutation,
+        $(D isPermutation([1, 2], [2, 1])) returns $(D true).)
+)
```

Comment:
Just noticed, these need to go before the parentheses. They aren't getting into the docs.

Also need to update the cheat sheet in std.algorithm itself (I think).

I'm satisfied with this addition after these fixes. Great work, thanks for sticking with it!